### PR TITLE
feat: rating and toast component added

### PR DESCRIPTION
**Pull request by devdeadviz**
# Change Logs
## Screenshots
Screenshot (142)
Screenshot (143)
## Deploy Preview Link:
Rating: https://deploy-preview-8--arcui.netlify.app/pages/components/rating/rating.html
Toast: https://deploy-preview-8--arcui.netlify.app/pages/components/toast/toast.html

### CODE REVIEW

File: pages/components/rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+@import "../../../styles/margin.css";
+@import "../../../styles/utils.css";
+
+.simple-rating-wrapper span {
+  cursor: pointer;
+}
+
+.checked {
+  color: orange;
+}
```
Comment (mohitdhatrak) [inline pages/components/rating/rating.css]: maybe use colours in hexcode, i got this suggestion so sharing. If i am wrong ignore it :)

File: pages/components/toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+.toast-wrapper {
+  background: rgb(50, 50, 50);
+  padding: 1rem;
+  border-radius: 12px;
+  width: 100%;
+  max-width: 700px;
+}
+
+.toast-text {
+  color: var(--white-color);
```
Comment (mohitdhatrak) [inline pages/components/toast/toast.css:19]: don't use color names in the variable, if you want to change color then you have to change name also. Use generic names like text-color or toast-color

File: pages/components/toast/toast.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  font-weight: lighter;
+}
+
+.toast-btn {
+  position: relative;
+  background: transparent;
```
Comment (mohitdhatrak) [inline pages/components/toast/toast.css:25]: Why position relative needed? Just use flex properties right. Mine worked that way. Just a suggestion

File: pages/components/toast/toast.html [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+    <div class="flex">
+      <!-- Sidebar -->
+      <aside class="drawer-wrapper">
+        <div class="drawer-header">
+          <h2 class="drawer-heading">Get Started</h2>
+        </div>
+        <hr />
+        <div class="drawer-options">
+          <a
+            class="drawer-lists flex flexAlignItemsCenter"
```
Comment (mohitdhatrak) [inline pages/components/toast/toast.html:62]: I guess i had pointed this last time, why is one class name without hyphen and camel case. If there is a reason then np

**Review by mohitdhatrak — COMMENTED** [review]
Good work kuldeep! Few small suggestions, have a look.

File: pages/components/rating/rating.css (hunk 1 above)
Comment (devdeadviz) [reply pages/components/rating/rating.css]: Yeah sure.

File: pages/components/rating/rating.css [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+@import "../../../styles/drawer.css";
+@import "../../../styles/padding.css";
+@import "../../../styles/margin.css";
+@import "../../../styles/utils.css";
+
+.simple-rating-wrapper span {
+  cursor: pointer;
+}
+
+.checked {
```
Comment (xii-vi) [inline pages/components/rating/rating.css:14]: what is checked here??

File: pages/components/toast/toast.css (hunk 2 above)
Comment (devdeadviz) [reply pages/components/toast/toast.css:19]: Used for specific purpose.

File: pages/components/rating/rating.html [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+        <img class="logo" src="../../../assets/logo.png" alt="Arc UI Logo" />
+        <h1><a class="header-heading" href="../../../index.html">Arc UI</a></h1>
+      </div>
+      <div class="header__nav">
+        <nav>
+          <ul>
+            <li>
+              <a
+                target="_blank"
+                class="social-icons-wrapper"
```
Comment (xii-vi) [inline pages/components/rating/rating.html:24]: social-media-icon would be more appropriate, just saying

File: pages/components/toast/toast.css (hunk 3 above)
Comment (devdeadviz) [reply pages/components/toast/toast.css:25]: Okay will try that.

File: pages/components/toast/toast.html (hunk 4 above)
Comment (devdeadviz) [reply pages/components/toast/toast.html:62]: I have used camel case for layouts just to make it differentiated from other classes.

File: pages/components/rating/rating.css (hunk 5 above)
Comment (devdeadviz) [reply pages/components/rating/rating.css:14]: I will change that name.

**Review by xii-vi — APPROVED** [review]
Few Comments are there, resolve them.
After that, you can merge.

File: pages/components/rating/rating.html (hunk 6 above)
Comment (devdeadviz) [reply pages/components/rating/rating.html:24]: Okay

**Pull request merged**